### PR TITLE
add initial docs

### PR DESCRIPTION
**Pull request by mathause**
- [ ] towards #29

Now that #28 is merged I can add an initial PR adding documentation. Details, as always, up for debate. To see the documentation locally you need to install some dependencies (I might miss some - let me know), build it and you can then check it out on the browser:

```bash
mamba install sphinx_rtd_theme sphinx sphinx-book-theme numpydoc
cd doc
make html
firefox build/html/index.html
```

### CODE REVIEW

File: doc/source/index.rst [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+.. toctree::
+   :maxdepth: 2
+   :hidden:
+   :caption: For users
+
+   installation
+   api
+
+**MESMER** - Modular Earth System Model Emulator with spatially Resolved output.
```
Comment (mathause) [inline doc/source/index.rst:17]: @leabeusch do you want to write 1-2 high level sentences what mesmer is and can do? See e.g. http://xarray.pydata.org/en/stable/index.html.

**Review by mathause — COMMENTED** [review]
I suggest to merge this more or less as is. I can then open a new PR an try to get this working on RTD. If someone wants to have a look go ahead - else I'll use my admin privileges ;)

@leabeusch: if you want you can open an account on https://readthedocs.org/accounts/signup/ (I used "Sign up with Github")

File: doc/source/index.rst (hunk 1 above)
Comment (leabeusch) [reply doc/source/index.rst:17]: Tried in #77 now.

**Review by leabeusch — APPROVED** [review]
Fantastic, thanks a lot for all of this @mathause! Bit confused about what went wrong with my PR though because the function that you had to remove at the end was exactly the same function I had already removed as part of my PR? I guess my attempt to mimic your merging approach in PRs still has some room for improvement. :sweat_smile:

**Pull request merged**